docs/docs/stories.mdx

```diff
@@ -185,3 +187,64 @@ into a single file for evaluation. Read more about this format in [Testing Your
 This format is only used for testing and cannot be used for training.
 
 :::
+
+
+## End-to-end Training
+
+:::caution experimental feature
+End-to-end training is an experimental feature.
+We introduce experimental features to get feedback from our community, so we encourage you to try it out!
+However, the functionality might be changed or removed in the future.
+If you have feedback (positive or negative) please share it with us on the [Rasa Forum](https://forum.rasa.com).
+
+:::
+
+With end-to-end training, you do not have to deal with the specific
+intents of the messages that are extracted by the NLU pipeline
+or with separate `utter_` templates in the domain file.
+Instead, you can simply put the texts of the user and/or bot messages directly in the stories.
+See the [training data format](./training-data-format.mdx#end-to-end-training-stories)
+for detailed description of how to write end-to-end stories.
+
+Rasa end-to-end training is fully integrated with standard Rasa approach.
+It means that you can have mixed stories with some steps defined by actions
+and other steps defined by user or bot utterance directly.
+
+We call it end-to-end training because actual text is fed into policies.
+
+:::note nlu is ignored during e2e
+For end-to-end user inputs, intents classified by the NLU pipeline
+and extracted entities are ignored.
```

Comment:
I'd add that they should still provide NLU training data. Users might otherwise think they can ignore NLU completely.

Reply:
what do you mean? if they don't have intents in the stories, they don't need NLU data

Reply:
Mhm, but isn't the `CountVectorizer`'s vocabulary relatively small (unless they have very very many end-to-end stories)? Having NLU training data seems the easier way to provide the data to me?

Reply:
not sure I understand what do you mean? the fact that you'll increase `CVF`'s vocabulary with NLU data will not affect TEDPolicy training

Reply:
NLU and e2e core is effectively independent, the only reason we reuse featurization is to avoid duplication, so data in NLU doesn’t influence e2e core model and data in e2e core doesn’t influence NLU model.